src/spead2_send.cpp

```
@@ -243,9 +249,6 @@ int main(int argc, const char **argv)
     options opts = parse_args(argc, argv);
 
     spead2::thread_pool thread_pool(opts.threads);
-    udp::resolver resolver(thread_pool.get_io_service());
-    udp::resolver::query query(opts.host, opts.port);
-    auto it = resolver.resolve(query);
```

Removing this is going to make the SPEAD2_USE_IBV codepath fail to compile. Also if `--ibv` and `--tcp` are both given the tool should report an error to the user.